creator-node/src/snapbackSM.js

```diff
@@ -157,7 +157,7 @@ class SnapbackSM {
   }) {
     try {
       const primaryClockValue = (await this.getUserPrimaryClockValues([userWallet]))[userWallet]
-      return this.issueSecondarySync({
+      return this.enqueueSecondarySync({
```

Comment:
RE:Above let's avoid unnecessary scope creep like this

Reply:
removed